scripts/utils/bdlClassnameManager

```diff
@@ -211,7 +218,7 @@ async function main() {
                 .join('|'),
             'g',
         );
-    } else if (fileType === 'js') {
+    } else if (jsExtensions.has(fileType) || fileType === '') {
```

Comment:
Why `|| fileType === ''` ?

Reply:
Good catch. I was originally adding `|| fileType === 'tsx'` to all the `if` statements, so I must have started to do that here and then forgotten to remove the extra clause.